### ISSUE TITLE
Feature Request: Dark Mode

### ISSUE THREAD

**Issue by arena** [ProgressiveCoders/Spoke#1368]
**Problem**
Sometimes my eyes hurt and I want to conserve electricity while I give them a rest. 

**Solution**
Dark Mode
sub-features
- way to toggle to dark mode
- user setting: 
     - light mode
     - dark mode
     - automatic based on time of day
     - automatic based on system setting
- light to dark color map

**Design Note**
Rather than inverting the light mode the colors should be dark but slightly lower contrast. I will add a mockup. ...need to take some inspiration from other dark mode apps and experiment a little to get the right colors.

**Comment by azuzunaga** [comment]
Great idea, @arena! I'd add `based on user OS preference` to the `user setting` list.

**Comment by arena** [comment]
@azuzunaga Great suggestion! Is that something that is possible to do with a web app (as opposed to a native mobile app).

**Comment by azuzunaga** [comment]
Yes! See support for the feature in web browsers here: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

**Comment by arena** [comment]
@stefanhayden is there a way to do this via a Material UI theme now that Material UI is updated?

**Comment by stefanhayden** [comment]
@arena yes. I've actually done most of the work already. we just need a UI for users and dark mode should mostly just work.
